appinventor/components/src/com/google/appinventor/components/runtime/Image.java

```diff
@@ -37,7 +37,7 @@
 @DesignerComponent(version = YaVersion.IMAGE_COMPONENT_VERSION,
     category = ComponentCategory.USERINTERFACE,
     description = "Component for displaying images.  The picture to display, " +
-    "and other aspects of the Image's appearance, can be specified in the " +
+    "and other aspects of the image's appearance, can be specified in the " +
```

Comment:
I would leave this as **Image** with a capital as it is referring to the component (noun).